arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts

```diff
@@ -64,8 +64,12 @@
 	aliases {
 
 #ifdef EEPROM_PROG_ENABLE
-		i2c100 = &channel_0_0;
+		i2c100 = &p0_channel_0_0;
+		i2c200 = &p1_channel_0_0;
 #endif
+		i2c102 = &p0_channel_0_2;
+		i2c202 = &p1_channel_0_2;
```

Comment:
to maintain the bus numbers in ascending order, move i2c202 after i2c123.

Reply:
Ack